backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/DockerCassandraSingleton.java

```diff
@@ -35,8 +35,8 @@ public static void incrementTestsPlayed() {
     }
 
     // Call this method to ensure that cassandra is restarted every MAX_TEST_PLAYED tests
-    public static void restartAfterMaxTestsPlayed() {
-        if (testsPlayedCount > MAX_TEST_PLAYED) {
+    public static void restartAfterMaxTestsPlayed(boolean forceRestart) {
```

Comment:
I doubt you want to take a forceRestart in this method. That's two different purpose, you should write a new method `restartIfNeeded`